packages/manager/modules/pci/src/projects/project/notebooks/translations/Messages_fr_FR.json

```diff
@@ -1,3 +1,23 @@
 {
```

Comment:
This file contains english sentences

Reply:
We dont have the content as yet. Sorry for not mentioning it earlier